### DISCUSSION TITLE
Property 'icons' does not exist on type

### DISCUSSION THREAD

**Discussion by rodzy** [chakra-ui/chakra-ui#2549] [Q&A] [answered]
Hi, everyone 😁

I've been using Chakra for a while but I always ditch the icons property from `chakraTheme` in TypeScript , is the any work around to get type safety on this property? 🤔 

![image](https://user-images.githubusercontent.com/49137701/99422964-aeb99b80-28c5-11eb-9701-45229ee9bdec.png)

**Comment by ljosberinn** [comment, marked as answer]
With v1, the icons are no longer part of the theme, [as mentioned in the docs](https://chakra-ui.com/docs/getting-started#optional-packages).

I'd also recommend using `extendTheme` instead of spreading the theme yourself :)
